.github/workflows/_build.yml

```diff
@@ -127,11 +127,11 @@ jobs:
 
   Screenshots:
     name: Screenshots tests
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
was hoping we can keep the same OS and keep snapshot test changes to a minimum, but unfortunately the gradle task wasn't running on macos - it didn't use the right jvm (i don't think that has anything to do with roborazzi, though). this matches the other non-emulator tasks